Added Jenkinsfile

dotIN opened the pull request:
* Added and tested the Jenkinsfile
* Updated metadat.rb

dotIN committed:
Refactored Bitbucket Install steps as Custom Resource

dotIN committed:
Refactored Bitbucket Install steps as Custom Resource

dotIN committed:
Refactored Bitbucket Install steps as Custom Resource

dotIN committed:
Refactored Bitbucket Install steps as Custom Resource

dotIN committed:
chef

dotIN committed:
chef

dotIN committed:
chef

dotIN committed:
chef

dotIN committed:
From upstream

dotIN committed:
Adding test steps and stages

dotIN committed:
Adding test steps and stages

dotIN committed:
Minor fixes in merge

dotIN committed:
Minor fixes in merge

bharathcp commented on metadata.rb:
```diff
@@ -1,12 +1,12 @@
-depends 'java', '~> 1.49.0'
-depends 'git', '~> 6.0.0'
+#depends 'java', '~> 1.49.0'
+#depends 'git', '~> 6.0.0'
```
without having these cookbooks as dependencies, the kitchen will fail. because java is part of the run list. without them the server will not start to perform integ test

Commit:
added food critic

removed FC009 since it was throwing an unnecessary error on
systemd_unit
fixed a new food critic issues

dotIN committed:
adding java and git depends

dotIN committed:
Merge pull request #5 from bharathcp/foodcritic

added food critic

dotIN committed:
update meta
